lib/types/CredentialIssuance.types.ts

```diff
@@ -76,29 +97,7 @@ export interface JWTPayload {
   exp?: number; // Not longer than 5 minutes
 }
 
-export interface JWTSignerArgs {
-  header: JWTHeader;
-  payload: JWTPayload;
-  privateKey: KeyObject;
-  publicKey: KeyObject;
-}
-
-export interface JWTVerifyArgs {
-  jws: string;
-  key: KeyObject;
-  algorithms?: Alg[];
-}
-
-export interface ProofOfPossessionOpts {
-  issuerURL?: string;
-  clientId?: string;
-  jwtSignerArgs: JWTSignerArgs;
-  jwtSignerCallback: JWTSignerCallback;
-  jwtVerifyCallback?: JWTVerifyCallback;
-}
-
-export type JWTSignerCallback = (args: JWTSignerArgs) => Promise<string>;
-
-export type JWTVerifyCallback = (args: JWTVerifyArgs) => Promise<void>;
+export type JWTSignerCallback = (args: { header?: unknown; payload?: unknown; [x: string]: unknown }) => Promise<string>;
```

Comment:
Why are these unknowns? I would expect something that resembles a JWT header and payload here